Marlin/Configuration.h

```diff
@@ -433,6 +433,64 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
 //#define Z_PROBE_SLED
 //#define SLED_DOCKING_OFFSET 5 // The extra distance the X axis must travel to pickup the sled. 0 should be fine but you can push it further if you'd like.
 
+// With this option servos are powered only during movement, then turned off to prevent jitter.
+//#define DEACTIVATE_SERVOS_AFTER_MOVE
+
+
+// With probe what is triggered when stowed we can test if it is deployed/stowed or not.
+// This is common for Allen Key probes and some servo probes.
+//#define PROBE_IS_TRIGGERED_WHEN_STOWED_TEST
+
+// Allen key retractable z-probe as seen on many Kossel delta printers - http://reprap.org/wiki/Kossel#Automatic_bed_leveling_probe
+// Deploys by touching z-axis belt. Retracts by pushing the probe down. Uses Z_MIN_PIN.
+//#define Z_PROBE_ALLEN_KEY
+// Allow homing with fake Allen key probe. A debug feature.
+#define TEST_ALLEN_KEY
```

Comment:
Perhaps it's a good time to use a single `PROBE_TYPE` option instead of all these others. We just need some pre-defined values, like…
```cpp
#define PROBE_TYPE_SERVO          100
#define PROBE_TYPE_FIXED          101
#define PROBE_TYPE_ALLEN_KEY      102
#define PROBE_TYPE_FAUXALLEN_KEY  202
#define PROBE_TYPE_SLED           103
```
Then users ony need to set one option…
```cpp
#define PROBE_TYPE PROBE_TYPE_SERVO
```